script/script.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-loop-func */
+/* eslint-disable no-unused-vars */
+/* eslint-disable linebreak-style */
+/* eslint-disable no-undef */

```

Comment:
Instead of disabling so many rules, try and address them. Maybe I can help. :slightly_smiling_face:

Reply:
It improves your coding style as well and helps you keep the right standards